src/js/core/widget/core/Popup.js

```diff
@@ -644,6 +697,8 @@
 				eventUtils.on(self._ui.page, "pagebeforehide", self, false);
 				eventUtils.on(window, "resize", self, false);
 				eventUtils.on(document, "vclick", self, false);
+
+				self._onScroll(self._ui.content);
```

Comment:
move this method to `prototype.handleEvent`
```
					case "scroll":
						self._onScroll(event);
						break;
```

Reply:
done.